yt_dlp/extractor/youtube.py

```diff
@@ -2409,18 +2412,10 @@ def _generate_comment_continuation(video_id):
     def _extract_comments(self, ytcfg, video_id, contents, webpage):
         """Entry for comment extraction"""
         def _real_comment_extract(contents):
-            if isinstance(contents, list):
-                for entry in contents:
-                    for key, renderer in entry.items():
-                        if key not in known_entry_comment_renderers:
-                            continue
-                        yield from self._comment_entries(
-                            renderer, video_id=video_id, ytcfg=ytcfg,
-                            identity_token=self._extract_identity_token(webpage, item_id=video_id),
-                            account_syncid=self._extract_account_syncid(ytcfg))
-                        break
+            yield from self._comment_entries(
+                traverse_obj(contents, (..., ('itemSectionRenderer',)), get_all=False), ytcfg, video_id)
```

Comment:
the extra tuple isn't needed, right?
```suggestion
                traverse_obj(contents, (..., 'itemSectionRenderer'), get_all=False), ytcfg, video_id)
```